client/src/components/reader/TocDrawer.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component, PropTypes } from 'react';
 import { Toc } from 'components/reader';
+import ScrollLock from 'components/global/Utility/ScrollLock';
 import classNames from 'classnames';
```

Comment:
You and I have a minor style inconsistency here. I've gotten in the habit of `import classnames from 'classnames';` whereas you tend to camelCase it. We should decide on one way or another to reduce minor cognitive dissonance. I prefer the former, because it lines up with the module name, but could go either way.